test/rubygems/test_gem_package_tar_reader_entry.rb

```diff
@@ -211,4 +211,19 @@ def test_zero_byte_file_readpartial
   ensure
     close_util_entry(zero_entry) if zero_entry
   end
+
+  def test_read_from_gzip_io
+    tgz = util_gzip(@tar)
+
+    Zlib::GzipReader.wrap StringIO.new(tgz) do |gzio|
+      begin
```

Comment:
:thinking: Any reason to wrap this into `begin` block?

Reply:
Oops, leftover from an `ensure` that was used in the first try at writing this test. I'll fix.